examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -419,7 +261,10 @@ decode_results capture;  // Somewhere to store inbound IR messages.
 MDNSResponder mdns;
 WiFiClient espClient;
 WiFiManager wifiManager;
-
+bool flagSaveWifiConfig = false;
+char HttpUsername[kUsernameLength + 1] = "admin";  // Default HTT username.
+char HttpPassword[kPasswordLength + 1] = "";  // No HTTP password by default.
+char Hostname[kHostnameLength + 1] = "ir_server";  // Default hostname.
```

Comment:
Are you sure you want to keep these here and not move them to `IRMQTTServer.h`? If yes, then maybe putting a reference to it next to `FIRMWARE_OTA ` and `HTML_PASSWORD_ENABLE ` would be useful I think. Just a comment like `Note: Firmware OTA is also disabled until a password is set. You can do this by updating HttpPassword  in IRMQTTServer.ino`

Reply:
Yes, I was keeping the global variables in the `.ino` file, rather than in the `.h` file.

Technically, the password (for OTA) is now set in the WifiManager initial boot/menu thing.
I could add a note/link to the reset page for it. There are instructions in the comment/docs at the start of the `.ino` file too.

Reply:
Made it clearer how to set the password, and make it clearer it's required for the OTA to work.